tests/divToField.spec.js

```diff
@@ -0,0 +1,46 @@
+import {expect} from 'chai';
+import fixugen from '@natlibfi/fixugen';
+
+//import {MarcRecord} from '@natlibfi/marc-record';
+//import createReducer from './postprocessor';
+import {READERS} from '@natlibfi/fixura';
+const {default: generateTests} = fixugen;
+
+import {filterField, stringToMarcField} from '../src/scripts/marcRecordUi.js';
+
+import {JSDOM} from 'jsdom';
+
+
+describe('html div->json field ', () => {
+  generateTests({
+    callback,
+    path: [import.meta.dirname, 'testFixtures', 'field'],
+    recurse: false,
+    useMetadataFile: true,
+    fixura: {
+      failWhenNotFound: false,
+      reader: READERS.TEXT
+    }
+  });
+
+  function callback({getFixture, decorator = {}}) {
+    const targetJson = JSON.parse(getFixture('field.json'));
+    //console.log(JSON.stringify(inputField));
+    const inputHtml = `<!DOCTYPE html><div id="editor">${getFixture('target.html')}</div>`.replace(/\n/gu, '');
```

Comment:
target.hml content is small, and could be added in metadata.json to minimize resource usage.

Reply:
done